### PR TITLE
allow selecting a different environment so we can deploy pod12 which …

### PR DESCRIPTION
**Pull request by grosser**
…is in pre-prod

@jonmoter 

screen shot 2016-01-20 at 10 59 18 am
screen shot 2016-01-20 at 10 59 08 am

https://zendesk.atlassian.net/browse/POD-236

### Risks
 - None

### CODE REVIEW

File: app/views/admin/deploy_groups/_environment_selector.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<%= form_tag '?', method: :get do %>
+  Environment: <%= select_tag :environment_id, options_for_select([['Matching', '']] + Environment.all.map { |e| [e.name, e.id] }, params[:environment_id]) %>
```
Comment (jonmoter) [inline app/views/admin/deploy_groups/_environment_selector.html.erb]: How about the default value is the environment of the deploy group?

**Comment by jonmoter** [comment]
:+1:

File: app/views/admin/deploy_groups/_environment_selector.html.erb (hunk 1 above)
Comment (grosser) [inline app/views/admin/deploy_groups/_environment_selector.html.erb]: Yeah true ... that's what matching was supposed to mean ... but we can be
clearer :)

Comment (grosser) [inline app/views/admin/deploy_groups/_environment_selector.html.erb]: updated

**Pull request merged**